lib/modules/platform/bitbucket/index.ts

```diff
@@ -68,13 +70,18 @@ export async function initPlatform({
   }
   setBaseUrl(defaults.endpoint);
   renovateUserUuid = null;
+  const options: InternalHttpOptions = {
+    useCache: false,
+  };
+  if (token) {
+    options.headers = { Authorization: `Bearer ${token}` };
```

Comment:
```suggestion
    options.token = token;
```

our http api should handle the header for you

Reply:
The `token` property doesn't seem to exist on the [InternalHttpOptions interface](https://github.com/renovatebot/renovate/blob/main/lib/util/http/types.ts#L70). Let me know if I should take a different strategy

Reply:
I'll add it to `HttpOptions` interface on seperate PR

- #16967